### PR TITLE
mon: add mon tick completed latency

### PR DESCRIPTION
**Pull request by YiteGu**
## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "quincy"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

- When filling out the below checklist, you may click boxes directly in the GitHub web UI.  When entering or editing the entire PR message in the GitHub web UI editor, you may also select a checklist item by adding an `x` between the brackets: `[x]`.  Spaces and capitalization matter when checking off items this way.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [x] New feature (ticket optional)
  - [ ] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [ ] Updates relevant documentation
  - [x] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
- `jenkins test rook e2e`
</details>

### CODE REVIEW

**Comment by YiteGu** [comment]
jenkins test make check

**Review by Matan-B — APPROVED** [review]

File: src/mon/Monitor.cc [hunk 1]
```diff
@@ -5983,6 +5983,9 @@ void Monitor::tick()
   }
 
   mgr_client.update_daemon_health(get_health_metrics());
+
+  const utime_t lat = ceph_clock_now() - now;
+  dout(11) << " tick completed, latency: " << lat << dendl;
```
[the remark this answers is not in the document]
Comment (Matan-B) [reply src/mon/Monitor.cc]: @YiteGu, My bad here, I have meant either 10 or 20.

**Review by Matan-B — APPROVED** [review]

**Comment by YiteGu** [comment]
jenkins test api

File: src/mon/Monitor.cc [hunk 2]
```diff
@@ -5887,7 +5887,7 @@ void Monitor::new_tick()
 void Monitor::tick()
 {
   // ok go.
-  dout(11) << "tick" << dendl;
+  dout(10) << "tick begin" << dendl;
```
Comment (batrick) [inline src/mon/Monitor.cc:5890]: I'd prefer we use the `ceph::coarse_mono_clock` for this and any new purpose. (No need to update the `now` variable below, just make a new second one.)

**Review by batrick — CHANGES_REQUESTED** [review]

**Pull request closed without merging**